app/assets/javascripts/logging_utils.js.coffee

```diff
@@ -9,7 +9,9 @@ class @Logger
     if @enable_rationale and result? and typeof(result.isTrue) == 'function'
       if result.isTrue() and result.length
         json_results = _.map(result,(item) -> {id: item.id, json: item.json})
-        @rationale[id] = {results: json_results }
+        if typeof result.specificContext != 'undefined'
```

Comment:
A more coffeescriptish way to check this would be

    if result.specificContext?

It's not precisely the same, the JS it produces is

    if (result.specificContext != null)

but I think it should serve?